### ISSUE TITLE
Error occurs when importing skimage with numpy==1.16.0

### ISSUE THREAD

**Issue by dongxiexiyin** [numpy/numpy#12732]
When importing the model _skimage_ with the latest numpy 1.16.0, following error occurs:

### Error message:

```
Python 2.7.14 (default, Jan  4 2019, 09:38:34) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-11)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import skimage
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/work/waifu-python/lib/python2.7/site-packages/skimage/__init__.py", line 167, in <module>
    from .util.dtype import (img_as_float32,
  File "/home/work/waifu-python/lib/python2.7/site-packages/skimage/util/__init__.py", line 6, in <module>
    from .apply_parallel import apply_parallel
  File "/home/work/waifu-python/lib/python2.7/site-packages/skimage/util/apply_parallel.py", line 8, in <module>
    import dask.array as da
  File "/home/work/waifu-python/lib/python2.7/site-packages/dask/array/__init__.py", line 9, in <module>
    from .routines import (take, choose, argwhere, where, coarsen, insert,
  File "/home/work/waifu-python/lib/python2.7/site-packages/dask/array/routines.py", line 256, in <module>
    @wraps(np.matmul)
  File "/home/work/.pyenv/versions/2.7.14/lib/python2.7/functools.py", line 33, in update_wrapper
    setattr(wrapper, attr, getattr(wrapped, attr))
AttributeError: 'numpy.ufunc' object has no attribute '__module__'
```
This issue is not reproducible in numpy 1.15.4.

### Numpy/Python version information:

Python: 2.7.14
NumPy: 1.16.0
scikit-image: 0.14.1

**Comment by eric-wieser** [comment]
This backtrace comes from dask, not from numpy

**Comment by eric-wieser** [comment]
Fixed in dask/dask#4346
